packages/flutter/lib/src/cupertino/icons.dart

```diff
@@ -143,4 +143,76 @@ class CupertinoIcons {
 
   /// Two right-facing intertwined arrows.
   static const IconData shuffle_thick = IconData(0xf221, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// Symbolizes a photo camera
```

Comment:
This particular description isn't very helpful... it's an "icon", or "symbol" of a "photo_camera", so it had better "symbolize a photo camera"... but what does that mean? When should the icon be used?